src/lib/components/workflow-actions.svelte

```diff
@@ -92,22 +95,33 @@
   width={200}
   text="You do not have permission to edit this workflow. Contact your admin for assistance."
 >
-  <SplitButton
-    primaryActionDisabled={cancelInProgress}
-    disabled={actionsDisabled}
-    label="Request Cancellation"
-    on:click={showCancellationModal}
-    id="workflow-actions"
-  >
-    <MenuItem
-      destructive
+  <FeatureGuard enabled={false}>
```

Comment:
enabled={!cancelDisabled}

Reply:
Maybe this should be cancelEnabled

Reply:
Lol was testing and forgot to revert, thx.